src/test/java/wolox/training/repositoriesTest/BookRepositoryTest.java

```diff
@@ -0,0 +1,81 @@
+package wolox.training.repositoriesTest;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.data.domain.Page;
+import org.springframework.test.context.junit4.SpringRunner;
+import wolox.training.entitiesTest.EntitiesTest;
+import wolox.training.models.Book;
+import wolox.training.repositories.BookRepository;
+
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@RunWith(SpringRunner.class)
```

Comment:
This annotation is not necessary with junit 5

Reply:
I removed this annotation.